avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -474,6 +453,83 @@ cpdef double computeResForce(double hRes, double h, double areaPart, double rho,
   return cRecResPart
 
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
+@cython.cdivision(True)
+cdef (double, double, double) addArtificialViscosity(double m, double h, double dt, double rho,
+                                                     double ux, double uy, double uz, double subgridMixingFactor,
+                                                     int Lx0, int Ly0, double w0, double w1, double w2, double w3,
+                                                     double[:, :] VX, double[:, :] VY, double[:, :] VZ,
+                                                     double nx, double ny, double nz):
+  """ add artificial viscosity
```

Comment:
add a bit more info here, e.g. that this is the implicit method as mentioned later in the function